config/rbac/kustomization.yaml

```diff
@@ -13,6 +13,6 @@ resources:
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
 # - auth_proxy_service.yaml
-# - auth_proxy_role.yaml
-# - auth_proxy_role_binding.yaml
+- auth_proxy_role.yaml
+- auth_proxy_role_binding.yaml
 # - auth_proxy_client_clusterrole.yaml
```

Comment:
This seems like something that users would want too.

Reply:
Yes, we definitely need that configure the kube-rbac-proxy correctly.  @francostellari is already including them in this PR.

Reply:
So you are going to do this?

Reply:
@MikeSpreitzer I already uncommented the SVC but @dumb0002 explained that you also wanted the cluster role line, so I uncommented that as well in the latest commit